### ISSUE TITLE
Section 508 502.3.1: (dotnet/list 'Controls list') The Name property must not include the element's control type.: A11y_Winform_ListTreeGrid_AI4D

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/winforms#15174]
### Github Tags:
#A11yTCS; #A11yMAS; #WCAG4.1.2; #A11ySev3; #E2E_WinForms.NETRuntime_Win32_Sept2026; #DesktopApp; #FTP; #WinForms .NET Runtime; #NameRoleValue; #A11yAuto; #AI4D; #a11ydb-button-name-extra;

### Issue Subcategory:
Button name has extra information

### User Experience:
Overly verbose names make it harder to tell which part of the announcement matters, degrading the listening experience as people either need to interrupt the screen reader repeatedly or sit through excessive announcements. For people who use voice control to activate a control by speaking its accessible name, a long name can make the control harder to target.

### Repro Steps:
Launch Winforms app > List, grid and tree > Observe the issue.

**App:** dotnet

**Element path:** 
list 'Controls list'

**Issue Details:** 
The Name property must not include the element's control type. [Section 508 502.3.1](https://www.access-board.gov/ict/#502-interoperability-assistive-technology)

**How To Fix:** 
Provide a UI Automation Name property for the element that:
 · Concisely identifies the element, AND
 · Does not include the control type.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

### Environment details:
Application Name: WinForms .NET Runtime
OS: Windows 11 Enterprise Insider Preview 26300.8276

### Note:
"Contact [CAI Accessibility Support Channel](https://teams.microsoft.com/l/channel/19:DeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%40thread.tacv2/CAI%20Accessibility%20Support%20Channel?groupId=f5eb428c-9e6e-4c16-8143-33fe3961a9ae&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) for Accessibility queries on design, development, test, usability or any processes"
Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Attachment:

<img width="1361" height="699" alt="Image" src="https://github.com/user-attachments/assets/a26d87a1-beb3-4e56-b080-1a4ad160122d" />

**Comment by LeafShi1** [comment]
The `AccessibleName` can be customized by the application. If the control type is coming from the configured `AccessibleName`, this doesn't appear to be a WinForms issue. The `AccessibleName` should be updated so that it doesn't include the control type.
